com.unity.netcode.gameobjects/TestHelpers/Runtime/NetcodeIntegrationTest.cs

```diff
@@ -1342,6 +1344,13 @@ protected IEnumerator CoroutineShutdownAndCleanUp()
         {
             VerboseDebug($"Entering {nameof(ShutdownAndCleanUp)}");
             // Shutdown and clean up both of our NetworkManager instances
+
```

Comment:
You don't need this as it is handled at the end of this coroutine:

            // Assure any remaining NetworkManagers are destroyed
            DestroyNetworkManagers();